adapters/Vungle/VungleAdapter/GADMAdapterVungleRouter.m

```diff
@@ -27,7 +27,7 @@ @implementation GADMAdapterVungleRouter {
   NSMapTable<NSString *, id<GADMAdapterVungleDelegate>> *_initializingDelegates;
 
   /// Map table to hold the banner ad delegates with placement ID as a key.
-  NSMapTable<NSString *, id<GADMAdapterVungleDelegate>> *_bannerDelegates;
+  NSMapTable<GADMAdapterVungleBannerRequest *, NSMutableOrderedSet<id<GADMAdapterVungleDelegate>> *> *_bannerDelegates;
```

Comment:
The previous implementation is that one placementID only corresponds to one delegate, which could lead the refresh request delegate cannot be added to `_bannerDelegates`, after the old delegate is removed from `_bannerDelegates`, the `_bannerDelegates` is empty.

Reply:
Got it. Sounds good.